benchmark_runner/common/ocp_resources/local_storage/template/04_subscription_template.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   name: local-storage-operator
   namespace: openshift-local-storage
 spec:
-  channel: "{{ ocs_version }}"
+  channel: "{{ odf_version }}"
   installPlanApproval: Automatic
   name: local-storage-operator
-  source: redhat-operators  # <-- Modify the name of the redhat-operators catalogsource if not default
+  source: redhat-operators-v49  # <-- Modify the name of the redhat-operators catalogsource if not default
```

Comment:
Where does the "-v49" come from?

Reply:
-v49 is the channel the 4.9 builds got moved to recently, no idea why but we got this info from QE.. Once we move to the next release this will change.